src/app/app.component.scss

```diff
@@ -1,5 +1,5 @@
 .example-fill-remaining-space {
-  /* This fills the remaining space, by using flexbox. 
+  /* This fills the remaining space, by using flexbox.
        Every toolbar row uses a flexbox row layout. */
   flex: 1 1 auto;
 }
```

Comment:
useless, right?

Reply:
Not completely, otherwise the menu icon would not be on the far right. But there are neater ways to do this 😉  I leave it as it is not really touched by this PR.